cardano-db/test/test-db.hs

```diff
@@ -1,18 +1,37 @@
 {-# LANGUAGE OverloadedStrings #-}
 
+import           Prelude
+
 import           Test.Tasty (defaultMain, testGroup)
 
 import qualified Test.IO.Cardano.Db.Insert
 import qualified Test.IO.Cardano.Db.Migration
 import qualified Test.IO.Cardano.Db.TotalSupply
 import qualified Test.IO.Cardano.Db.Rollback
 
+import           System.Environment (lookupEnv, setEnv)
+import           System.Directory (getCurrentDirectory)
+import           System.FilePath ((</>))
+
 main :: IO ()
-main =
+main = do
+  -- If the env is not set, set it to default.
+  maybePgPassFile <- lookupEnv "PGPASSFILE"
```

Comment:
This, and the `whenNothing` should be a separate function. that is called just before `defaultMain`.